docs/antora/modules/ROOT/pages/setup/source.adoc

```diff
@@ -303,7 +305,7 @@ It is strongly recommended that you synchronize your Lisk Core node with the net
 
 If you have not already done so, clone the {url_github_service}[lisk-service^] GitHub repository and then navigate into the project folder and check out the latest release.
 
-[source,bash]
+
 ----
 # Clone Lisk Service repository
 git clone https://github.com/LiskHQ/lisk-service.git
```

Comment:
The source needs an update. Same suggestion for this as mentioned above for the `setup/docker.adoc`.